README.md

```diff
@@ -187,7 +187,24 @@ Next, provide the path to the `subnet-evm` repository and the path to a writeabl
 ./scripts/local/e2e_test.sh --local --subnet-evm ./subnet-evm --data-dir ~/tmp/e2e-test
 ```
 
-## ABI Bindings
+#### Run the E2E tests on another network
+
+The E2E tests can be run on another network by implementing the `Network` interface in `package network`. For example, the type `FujiNetwork` in `example_fuji_network.go` implements this interface, pointing to the [Amplify](https://subnets-test.avax.network/amplify), [Bulletin](https://subnets-test.avax.network/bulletin), and [Conduit](https://subnets-test.avax.network/conduit) Fuji subnets. After implementing this interface, you can run the E2E tests on this network by running a program such as:
```

Comment:
Is it worth linking `pakcage network` too?

Reply:
and/or `example_fuji_network`

Reply:
Added 👍